doc_source/resolver-rules-managing.md

```diff
@@ -179,4 +179,18 @@ Note the following:
 
 1. Choose **Delete**\.
 
-1. To confirm that you want to delete the rule, enter the name of the rule and choose **Submit**\.
\ No newline at end of file
+1. To confirm that you want to delete the rule, enter the name of the rule and choose **Submit**\.
+
+## Adding rules for reverse lookup<a name="add-reverse-lookup"></a>
+
+If you need to control reverse lookups in your VPC, you can add rules to your outbound resolver endpoint.
+
+### To create the reverse lookup rule
+
+ + Follow the procedure under [Creating Forwarding Rules](#resolver-rules-managing-creating-rules), up to step 5.
+ + When you specify your rule, enter the PTR record for the IP address you want a reverse lookup forwarding rule for.
+    + For example, if you need to forward lookups for addresses in the 10.0.0.0/23 range, enter a rules:
+      + 0.0.10.in-addr.apra
+      + 1.0.10.in-addr.apra
+  + Specify the server to forward these lookups to
```

Comment:
Missing a period at the end of this sentence.

Reply:
Hey thanks! I’d completely forgotten about this. I notice before the example at the end I’ve put ‘enter a rules’. ‘add rules’ or ‘enter rules’ would probably be better. I’m phoneposting atm but can take a look later.

Reply:
Were you able to take another look at this? Again, I'm happy to just make the changes myself by building on top of your commit, if that's easier for you.

Reply:
OK, just pushed some updates. FWIW, there seems to be bunch of trailing spaces on the headings in this file, which VS Code keeps complaining about.

Reply:
Yeah, sorry about the trailing spaces. Our authoring and publishing system sometimes adds those.

This looks good! I'll go ahead and merge the PR. I still want to move these steps up closer to the steps for creating forwarding rules, but I'll do that after the merge.

Thanks for your PR!